include/mega/megaclient.h

```diff
@@ -1389,6 +1389,7 @@ class MEGA_API MegaClient
     // remove node subtree
     void deltree(handle);
 
+    Node* nodeByHandle(NodeHandle) const;
```

Comment:
I think we should make a comment to the getter with `handle`:
```suggestion
    Node* nodeByHandle(NodeHandle) const;
    // obsolete, use version with NodeHandle as input param
```